src/pages/UserPreferences/RegistryPassword/__tests__/index.spec.tsx

```diff
@@ -98,8 +98,8 @@ describe('Registry Password Input', () => {
 
       const input = screen.getByTestId('registry-password-input');
 
-      const message = 'The password is too long. The maximum length is 100 characters.';
-      const allowedPassword = 'a'.repeat(100);
+      const message = 'The password is too long. The maximum length is 10000 characters.';
```

Comment:
such long limit suggest there is no a limit.
I would like to see that logic removed if it's relatively easy

@akurinnoy @olexii4 WDYT?

Reply:
The max length check on the client-side should be in any case.
As for me, 512 characters sounds better. But I do not see any problem with 1000.